docs/DeveloperGuide.md

```diff
@@ -168,6 +168,30 @@ The `SwitchCommand#execute()` returns the `CommandResult(MESSAGE_SUCCESS_GRADE,
 
 `GRADE_CHART` is an element in enumeration `ModelType` which determines what content is displayed in main screen.
 
+### Finding students
+
+
+The finding students feature, implemented by the `find` command, allows the user to see a filtered list of students.
+The filtering is based on an AND search, for example, `find n/John m/CS2103T` will show only students that have "John" in their name and are also from the CS2103T module.
+
+#### Implementation
+
+The following sequence diagram shows how the `find` command works.
+
+![FindSequenceDiagram](images/FindSequenceDiagram.png)
```

Comment:
I think you should resize diagrams so that the text size in the diagram matches the text size of the main text of the diagram. Currently, it's hard to see as the text in the diagram is quite small.

Reply:
How did you do it?

Reply:
I omitted some less important details. Another solution can be breaking diagrams into smaller fragments when possible